iris_grib/_load_convert.py

```diff
@@ -1380,6 +1380,13 @@ def translate_phenomenon(metadata, discipline, parameterCategory,
             metadata['long_name'] = long_name
             metadata['units'] = Unit(1)
 
+    # Add a standard attribute recording the grib phenomenon identity.
+    metadata['attributes']['GRIB_CODING'] = GribCode(
```

Comment:
I think `GRIB_CODING` lacks clarity on what it is referring to, and that it needs some reference to "parameter"
I would prefer something like
`GRIB_PARAM`
or 
`GRIB_PARAM_CODE`